### DISCUSSION TITLE
in basic networked-aframe users.

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#430] [General]
Hello, I was curious if it is possible to have little window boxes of each user showing there avatar and what color they are? a camera view of each user?

Joseph

**Comment by vincentfretin** [comment]
Hello, you can use multiple camera, each camera following a user.
This component may be of interest: https://github.com/diarmidmackenzie/aframe-multi-camera

**Reply by Therealjosephchrzempiec** [reply]
Hey Vincent, I'm having a hard time or understanding how can I attach this to all users who are on the scene so we can all see each others avatars?
